apps/src/weblab2/layout/VerticalLayout.tsx

```diff
@@ -7,20 +7,18 @@ import {LayoutProps} from '@codebridge/types';
 import HeaderButtons from '@codebridge/Workspace/HeaderButtons';
 import Workspace from '@codebridge/Workspace/Workspace';
 import classNames from 'classnames';
-import React, {useEffect, useState} from 'react';
+import React, {useEffect} from 'react';
 
 import {HTMLPreview} from '@cdo/apps/codebridge/FilePreview/HTMLPreview';
 import {useVerticalLayout} from '@cdo/apps/lab2/hooks/useVerticalLayout';
 import ResizeBar from '@cdo/apps/lab2/views/components/layout/ResizeBar';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
```

Comment:
I believe you need to `dispatch` the `setViewMode` redux action. Currently the toggle doesn't seem to be working.